installer.tpl.sh

```diff
@@ -122,6 +125,12 @@ setup_type ()
 
 setup_ver ()
 {
+  ARCH=$(uname -m)
+  if [ "${ARCH}" = "x86_64" ] && [ "${GC64:-OFF}" = "ON" ]; then
```

Comment:
I would propose to enable/disable GC64 via `true/false` like we do it at the tarantool repo. Otherwise, we have to remember one more approach for enabling GC64.

Reply:
Changed the value, now we can `export GC64=true` before installation to enable GC64 repo